changelog.md

```diff
@@ -12,13 +12,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - \[Demo\] Add new entries in about screen ([#403](https://github.com/Orange-OpenSource/ods-android/issues/403))
 - \[Lib\] Add `OdsFloatingActionButton` and `OdsExtendedFloatingActionButton` components ([#109](https://github.com/Orange-OpenSource/ods-android/issues/109))
 - \[Lib\] Add `OdsListItem` composable signature with an `OdsListItemTrailing` as trailing parameter in order to manage accessibility ([#387](https://github.com/Orange-OpenSource/ods-android/issues/387))
+- \[ThemeConfigurationContract\] Add `textFieldStyle` boolean in the theme contract to allow to choose between outlined or filled text fields in a custom theme ([#415](https://github.com/Orange-OpenSource/ods-android/issues/415))
```

Comment:
Should we also note that `OdsComponentCustomizations` has been renamed to `OdsComponentsConfiguration` ?